### ISSUE TITLE
[Question] Is this way to integrate firebase with django having Strangler pattern in main.    

### ISSUE THREAD

**Issue by romelgomez** [fcornelius/django-firebase-auth#3]
I have the task to migrate the django auth part 1.8~1.11 to firebase auth, with the Strangler Pattern https://docs.microsoft.com/en-us/azure/architecture/patterns/strangler has the main rule, the first thing that I think is make a custom provider for allauth django plugin. but don’t know is this is the right path to go.

Using the Strangler Pattern mean, that the User will login in the current app in production (django), but at the same time with firebase auth app in development, and vice versa. I have some advances, mostly in the firebase part.

The To-Do that I have:

[Done] Migrate the current users with the firebase-tools, Current users can login with the same password in firebase. https://github.com/firebase/firebase-tools/issues/1088

[ ] Login firebase and the same time in django and vice versa.

I have the idea that the custom provider will link the local and social accounts on django with the firebase custom provider, this part is not clear yet to me.

Basically I’m very noob with django and python. So I need some insights to get and define the right tasks for the next spring.
